theme/js/script.js

```diff
@@ -176,22 +182,97 @@ $.getUrlParameter = function(name){
 };
 
 function countCharacter() {
-    $('.text-area').on('input', function(){
+    $('.text-area').on('input', function () {
         var maxlength = $(this).attr('maxlength');
         var currentLength = $(this).val().length;
         var charCounter = $('.contact-form-container .character-counter');
 
-        if( currentLength >= maxlength ){
+        if (currentLength >= maxlength) {
             // If the current length of the text is 500 or more characters, display the specified text.
             charCounter.text('You have reached the maximum number of characters.');
-        } else{
+        } else {
             // If the length of the text is less than 500 characters, display the number of characters remaining.
             charCounter.text(maxlength - currentLength + ' characters remaining');
         }
 
-        if( currentLength >= 250) {
+        if (currentLength >= 250) {
             // Add class "limit-warning" to the character counter when there are 50 characters remaining or less.
             charCounter.addClass('limit-warning');
         }
     });
 }
+
+function openDonationTab(evt, donationType) {
+    var i, tabcontent, tablinks;
+
+    // Get all elements with class tabcontent and hide them.
+    tabcontent = document.getElementsByClassName('tabcontent');
+    for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = 'none';
+    }
+
+    // Get all elements with class tablinks and remove the active class.
+    tablinks = document.getElementsByClassName('tablinks');
+    for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(' active', '');
+    }
+
+    /* Show the current tab, and add an active class to the tab and the button that opened the tab.
+    donationType is a variable representing the monthly and single tabcontent ids.
+    This function needs this parameter passing through it. */
+    document.getElementById(donationType).style.display = 'block';
+    document.getElementById(donationType).className += ' active';
+    evt.currentTarget.className += ' active';
+
+
+    // Typing into the input field adds active class to the input field and removes the active class from other buttons.
+    $('.form-item-other .donate-form-input').on('input', function () {
+        $(this).addClass('active');
+        $('.donate-form-buttons .donate-sum').not('.donate-sum.active').removeClass('active');
+    });
+
+    // Clicking the donate button redirects to a thank you page.
+    $('.tabcontent .donate-button').click(function () {
+        var donationSum = $('.tabcontent.active .donate-sum.active').text();
+        var donationSumOther = $('.tabcontent.active .donate-form-input.active').val();
+
+        if (donationSum == null) {
+            window.location.href = '/thank-you-page.php?donationSum=' + donationSum + '';
```

Comment:
You, don't need the `+ ''` at the end Because you're just adding an empty string. Same for line 244

Reply:
done

Reply:
this breaks the js.